heartbeat/iSCSILogicalUnit.in

```diff
@@ -751,6 +786,11 @@ iSCSILogicalUnit_validate() {
 	tgt)
 		# tgt is userland only
 		;;
+	scst)
+		if [ ! -d /sys/kernel/scst_tgt ]; then
+			ocf_log err "/sys/kernel/scst_tgt does not exist or is not a directory -- check if required modules are loaded."
+			exit $OCF_ERR_INSTALLED
+		fi
```

Comment:
Can you finish it with `;;` here and in the other agent? It's just good practice to avoid the next dev adding a case where it could run through from the previous one.